### DISCUSSION TITLE
onProxyRes to handle websocket response

### DISCUSSION THREAD

**Discussion by wangwen666** [chimurai/http-proxy-middleware#761] [Q&A]
### Checks

- [X] I understand project setup issues should be asked on [StackOverflow](https://stackoverflow.com/questions/tagged/http-proxy-middleware) or in [GitHub Discussions](https://github.com/chimurai/http-proxy-middleware/discussions).
- [X] I updated to latest `http-proxy-middleware`.

### Describe the bug (be clear and concise)

I use onProxyRes to handle ws reponse, but it does not work,  onProxyRes can handler ws response? if i want to make it out,what should i do? thanks.

### Step-by-step reproduction instructions

```shell
/
```

### Expected behavior (be clear and concise)

/

### How is http-proxy-middleware used in your project?

```shell
/
```

### What http-proxy-middleware configuration are you using?

```typescript
/
```

### What OS/version and node/version are you seeing the problem?

```shell
/
```

### Additional context (optional)

/

**Comment by chimurai** [comment]
`onProxyRes` is for http traffic. You can try `onProxyReqWs`

**Reply by wangwen666** [reply]
thank you for your answer,  but you may have misunderstood me，i means， i want to handle ws response，but i doesnot find any ways to do, what should i do to handle ws response?

**Reply by chimurai** [reply]
http-proxy-middleware uses http-proxy under the hood.

The `on`-event are just aliases for http-proxy events:
https://github.com/http-party/node-http-proxy#listening-for-proxy-events

I don't see a `proxyResWs` in their list.
Best to ask your question on their issue tracker as a feature: https://github.com/http-party/node-http-proxy/issues
